### DISCUSSION TITLE
Regarding making an Android app

### DISCUSSION THREAD

**Discussion by Raj2032** [NativeScript/NativeScript#9719] [General]
![image](https://user-images.githubusercontent.com/29293830/147407935-7c83182a-faa2-4e8b-8733-981ac3c6abf9.png)

Hey guys,

I am looking to make a web based application for Android which can scan qr codes, image process it and extract information out of it. I have some questions regarding nativescript:

1. See the screenshot how there is a camera view in the middle of the screen and the rest is the app, would this be supported by nativescript?
2. In regards to the qr code image processing, is there a library out there that can be used with nativescript that can be done in real time while the camera is on?
3. Are there native android widgets available?
4. Which language/framework should I even use for my use cause? Angular or Vue?
5. Can I completely avoid using Android app studio?

**Comment by farfromrefug** [comment]
1/ yes
2/ you can use this https://github.com/nativescript-community/ui-barcodeview
3/ what do you mean?
4/ depends on what you like. You can even use svelte ;)
5/ yes

**Reply by Raj2032** [reply]
Ok thanks mate :)
